<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPrintCropPage</class>
 <widget class="QWidget" name="CropPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>325</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_disableCrop">
       <property name="whatsThis">
        <string>Do not crop photos, just scale them.</string>
       </property>
       <property name="text">
        <string>Do not crop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCropRotateLeft">
       <property name="toolTip">
        <string>Rotate photo to the left</string>
       </property>
       <property name="whatsThis">
        <string>Rotate photo to the left</string>
       </property>
       <property name="text">
        <string>&amp;Rotate left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCropRotateRight">
       <property name="toolTip">
        <string>Rotate photo to the right</string>
       </property>
       <property name="whatsThis">
        <string>Rotate photo to the right</string>
       </property>
       <property name="text">
        <string>&amp;Rotate right</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>127</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCropPrev">
       <property name="toolTip">
        <string>Previous photo</string>
       </property>
       <property name="whatsThis">
        <string>Previous photo</string>
       </property>
       <property name="text">
        <string>&lt;&lt; Pr&amp;evious</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCropNext">
       <property name="toolTip">
        <string>Next photo</string>
       </property>
       <property name="whatsThis">
        <string>Next photo</string>
       </property>
       <property name="text">
        <string>Ne&amp;xt &gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>127</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="LblCropPhoto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Photo 0 of 0</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
      <item row="0" column="0">
    <widget class="Digikam::AdvPrintCropFrame" name="cropFrame" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>201</height>
      </size>

     </property>

     <property name="whatsThis">
      <string>Move the box in order to crop photos so that they fit inside the photo sizes you have given.  
You can crop each image differently, or just click the 'Next' button to use the default crop 
setting for each photo.
Enable &quot;Do not crop&quot; to avoid cropping all of the photos.</string>
     </property>
    </widget>
   </item>
  <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="m_fileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSaveAs">
       <property name="text">
        <string>Save As</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item> 
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::AdvPrintCropFrame</class>
   <extends>QWidget</extends>
   <header>advprintcropframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
